File: .vscode/Java_Programs/Q9.java
//Problem Statement
//Write a Java program to check if a string is a palindrome.


import java.util.Scanner;

class Palindrome {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str, rev = "";
        System.out.println("Enter a string: ");
        str = sc.nextLine();

        for(int i = str.length(); i >= 0; i--) 
            rev += str.charAt(i);

        if(str.equals(rev))
            System.out.println("Palindrome");
        else
            System.out.println("Not a palindrome");

        sc.close();
    }
}

//Expected Output
        //Enter a string: madam
        //Palindrome

File: .vscode/C-Programs/Q9.c
//Problem Statement
//Write a program to find the length of a given string.

#include <stdio.h>

int main() {
    char str[50];
    int i, length = 0;
    printf("Enter a string: ");
    gets(str); 

    while(str[i] != '\0') { 
        length++;
        i++;
    }

    printf("Length of string: %d", length);
    return 0;
}

//Expected Output
        //Enter a string: hello
        //Length of string: 5

